quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/MetricsDistTest.java

```diff
@@ -60,9 +60,47 @@ void testMetricsEndpointNotEnabled(KeycloakDistribution distribution) {
     void testMetricsEndpoint() {
         when().get("/metrics").then()
                 .statusCode(200)
-                .body(containsString("jvm_gc_"))
-                .body(containsString("http_server_active_requests"))
-                .body(containsString("vendor_statistics_hit_ratio"))
+
+                // Test metrics used in Observability guides are present
+                // SLIs
+                .body(containsString("TYPE http_server_requests_seconds summary"))
```

Comment:
I see that this test fails. AFAIK this metric only appears on the second request, that's why I added a dummy request below in the next text case in this same file.

Reply:
Sorry I missed your comment and fixed by sending another metric request before assertion. Let me know whether it works for you or not.

Reply:
LGTM, this is what I had in mind. 

PS: I tend to comment on PRs that are in "ready to review state" :-)

Reply:
> PS: I tend to comment on PRs that are in "ready to review state" :-)

I did not expect this PR to fail so I did not send it as a draft, then I would do the green light for reviews by switching from draft to ready for review. I tried 100 repetitions locally. It is interesting the test was failing only when running in the container.